packages/fxa-payments-server/src/store/actions/index.test.ts

```diff
@@ -36,7 +21,7 @@ describe('resetActions', () => {
     'resetUpdateSubscriptionPlan',
   ];
   actionNames.forEach((name) => {
-    describe(name, () => {
+    describe(`${name}`, () => {
```

Comment:
A bit of personal curiosity here. 
What's the benefit of  `` ${name}`` versus just `name`?

Reply:
Just to get around the linter warning. Apparently the describe title needs to be a string, and `name` has a different type. So I figured this would be the easiest way to make this a string.